akka-projection-core/src/main/scala/akka/projection/ProjectionId.scala

```diff
@@ -4,10 +4,10 @@
 
 package akka.projection
 
-import java.util.stream.Collectors
-
 import scala.collection.immutable
 
+import akka.util.ccompat.JavaConverters._
```

Comment:
it's somewhat unfortunate to depend on Akka ineternals, but ok.
That was what I meant with that it will be difficult to avoid warnings for both 2.12 and 2.13, but ok let's use the akka ccompat

Reply:
I thought you suggested that at some point.